### PR TITLE
Assignment-3 | Router | Protected Router | useNavigate 

### PR DESCRIPTION
**Pull request by kaushalyad321**

### CODE REVIEW

File: Assignment-3/app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      <p className="quest">
+        4.Enhance the previous application by adding protected routes that
+        require authentication. Create a login page with a form that accepts a
+        username and password. Implement authentication logic (e.g., hardcode a
+        username and password for now). Secure the About page so that it can
+        only be accessed by authenticated users. Redirect unauthenticated users
+        to the login page. Display a message on the Home page welcoming the
+        authenticated user.
+      </p>
+      {/* <ProtectedRouter /> */}
```
Comment (sumitbaudh22) [inline Assignment-3/app/src/App.js:50]: Task 4 is commented, why?

File: Assignment-3/app/src/components/SuccessLogin.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const SuccessLogin = (props) => {
+  return (
+    <div>
+      <p>Welcome! {props.username}</p>
+    </div>
+  )
+}
```
Comment (sumitbaudh22) [inline Assignment-3/app/src/components/SuccessLogin.js]: What is the use of this component and where we are using it

File: Assignment-3/app/src/App.js (hunk 1 above)
Comment (kaushalyad321) [reply Assignment-3/app/src/App.js:50]: it is protected router when we will use then automatically we will move to login page of question-4
so other router will not work properly

File: Assignment-3/app/src/components/LogInAgain.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const LogIn = () => {
+  return (
+    <div>
+        
+      
+    </div>
+  )
+}
```
Comment (sumitbaudh22) [inline Assignment-3/app/src/components/LogInAgain.js]: Why do we have this component

File: Assignment-3/app/src/components/SuccessLogin.js (hunk 2 above)
Comment (kaushalyad321) [reply Assignment-3/app/src/components/SuccessLogin.js]: i can remove unwanted componnet

File: Assignment-3/app/src/components/LogInAgain.js (hunk 3 above)
Comment (kaushalyad321) [reply Assignment-3/app/src/components/LogInAgain.js]: no use of this componnet so i can remove it

File: Assignment-3/app/src/components/SuccessLogin.js (hunk 2 above)
Comment (sumitbaudh22) [reply Assignment-3/app/src/components/SuccessLogin.js]: Yes Please remove the unused files and component as it makes difficult to review

**Review by sumitbaudh22 — APPROVED** [review]

**Pull request merged**